README.md

```diff
@@ -19,6 +19,13 @@ Truffle is a development environment, testing framework and asset pipeline for E
 * Instant rebuilding of assets during development.
 * External script runner that executes scripts within a Truffle environment.
 
+### A Note To Developers
+
+The current active and stable version of Truffle can be found on the `master` branch.  There are two branches that are designated for developers to make pull requests against: the `develop` and `next` branches.
+
++   `develop`: Make your pull requests against the `develop` branch if you would like to make a change for the current major version.
```

Comment:
Also, since you expressed interest in improving your technical writing, let me dig from the bottom of the barrel of suggestions:

How about:
+ `develop`: For pull requests that must target the current major release
+ `next`: For pull requests that target the upcoming major release.

It might be also worth noting that targeting `next` is strongly preferred.

Reply:
Although this raises the question: will this information need to be updated right away when we get rid of `next`?

Reply:
(also, disclaimer: I claim no real expertise in matters of clarity. I just have enough experience being bad at writing things clearly that I've gotten ideas along the way... feel free to exercise judgment whenever.)